.github/workflows/changelog_verifier.yml

```diff
@@ -0,0 +1,22 @@
+name: "Changelog Verifier"
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review, labeled, unlabeled]
```

Comment:
Why not all PRs?

Reply:
The default mechanism for `pull_request` trigger is [as follows](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request): 

`By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened.`

I have updated the GHA config to reflect all the types I found could match our case.